dmn/flight-has-capacity.dmn

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<definitions xmlns="https://www.omg.org/spec/DMN/20230324/MODEL/" expressionLanguage="https://www.omg.org/spec/DMN/20230324/FEEL/" namespace="https://kie.org/dmn/_7785B66E-B0F4-473E-84DA-82D1EF89D006" id="_BD44F978-12D8-45E6-9E15-9EC830771AEB" name="DMN_AEC0C0CF-C716-491B-8B2F-F7AD62330916" xmlns:dmndi="https://www.omg.org/spec/DMN/20230324/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:kie="https://kie.org/dmn/extensions/1.0">
+  <itemDefinition id="_BB82C411-E1A1-4ABB-95FA-D48438AF09E8" name="tPassengerTable" isCollection="true" typeLanguage="https://www.omg.org/spec/DMN/20230324/FEEL/">
```

Comment:
`tPassengerTable` is a collection with `Flight Number`, I guess `tRebookedFlights` would be a better name? WDYT?

Reply:
will do, probably will need to update some screenshots in the related PR